pandas/tests/indexes/test_interval.py

```diff
@@ -458,6 +592,30 @@ def test_get_indexer_subintervals(self):
         expected = np.array([0, 0, 0], dtype='intp')
         tm.assert_numpy_array_equal(actual, expected)
 
+    @pytest.mark.xfail(reason="new indexing tests for issue 16316")
+    def test_get_indexer_subintervals_updated_behavior(self):
+
+        # a good chance this whole method needs reworking.
+
+        # target = IntervalIndex.from_breaks(np.linspace(0, 2, 5))
```

Comment:
BTW, this is the cause all CI fails, because you have a function definition here without content which results in a IndentationError for the next test